<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileSettings</class>
 <widget class="QWidget" name="ProfileSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>315</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="newProfileButton">
     <property name="toolTip">
      <string>Create a new profile based upon the selected profile</string>
     </property>
     <property name="text">
      <string>&amp;New Profile...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="editProfileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Edit the selected profile(s)</string>
     </property>
     <property name="text">
      <string>&amp;Edit Profile...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="deleteProfileButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Delete the selected profile(s)</string>
     </property>
     <property name="text">
      <string>&amp;Delete Profile</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="setAsDefaultButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Set the selected profile as the default for new terminal sessions</string>
     </property>
     <property name="text">
      <string>&amp;Set as Default</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTableView" name="sessionTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
